<?xml version="1.0"?>
<robot name="evbot">

   <material name="white">
     <color rgba="1 1 1 1"/>
   </material>

   <material name="yellow">
     <color rgba="0.97 0.8 0.2 1"/>
   </material>

  <link name="velodyne">

    <visual>
      <origin xyz="0.1 -4.35 -1.6" rpy="0 0 1.57" />
      <geometry>
        <mesh filename="package://hdl_localization/meshes/beetle.stl" scale="0.25 0.25 0.25" />
	<!--mesh filename="file:///home/ntu_yos/catkin_ws/src/hdl_localization/meshes/beetle.dae" scale="1 1 1" /-->
        <!--box size="2.5 1.2 1.5"/-->
      </geometry>
      <material name="yellow"/>
    </visual>

  </link>

</robot>
